types/applepayjs/index.d.ts

```diff
@@ -850,6 +874,46 @@ declare namespace ApplePayJS {
         tokenNotificationURL?: string;
     }
 
+    /**
+     * A dictionary that represents a request to set up a deferred payment, such as a hotel booking or a pre-order.
+     */
+    interface ApplePayDeferredPaymentRequest {
+        /**
+         * The localized billing agreement the framework displays to the user prior to payment authorization.
+         */
+        billingAgreement?: string | undefined;
+
+        /**
+         * A dictionary that contains details about the deferred payment.
+         */
+        deferredBilling: ApplePayLineItem;
+
+        /**
+         * [ISO 8601 formatted] The time zone at the destination location of the payment.
```

Comment:
I guess this should something like this. If the docs are wrong you'd have to ask Apple 😄

```suggestion
         * The date and time at the destination location of the payment.
```

Reply:
Suggested change applied, could you please take a look? 🙂